src/components/Settings/TextInput.tsx

```diff
@@ -1,14 +1,16 @@
-import { useState } from "react";
+import { Dispatch } from "react";
 
 export function TextInput({
+  description,
+  setDescription,
   textAbove,
   placeHolderText,
 }: {
+  description: string;
+  setDescription: Dispatch<string>;
   textAbove: string;
   placeHolderText: string;
 }) {
-  const [description, setDescription] = useState();
-
   const onTextChange = (e: any) => {
     if (e.target.value) setDescription(e.target.value);
```

Comment:
This makes it impossible to remove the last character from the text area, which is very frustrating.